reservation/tests.py

```diff
@@ -1,12 +1,10 @@
-import pytest
 from reservation.models import TypeOfJob, Appointment, Schedule
```

Comment:
Consider splitting this file into three different files
It is too long and contains different test objects

Reply:
Done